src/qe_tools/extractors/__init__.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+
+from qe_tools.exceptions import PathIsNotAFile
+from qe_tools.parsers import CpInputFile, PwInputFile
+
+
+def extract(filepath: str, parser: str) -> dict:
+    """Extract QE input file as a dictionary.
+
+    Parameters
+    ----------
+    `filepath` : `str`
+        The path to the file.
+    `parser` : `str`
+        The QE parser type.
```

Comment:
Should probably mention which values are supported. As far as I can tell, these are `PW` and `CP` and these are now _only_ hardcoded in this `extract` implementation.